config/environments/test.rb

```diff
@@ -5,6 +5,7 @@
 ENV['open_course_creation'] = 'false'
 ENV['course_prefix'] = 'Wikipedia:Wiki_Ed'
 ENV['wiki_language'] = 'en'
+ENV['wiki_education'] = 'false'
```

Comment:
This should be true.

Reply:
You can also remove the ENV values in this file that are no longer used in Features.

Reply:
I got more errors when I had that `true` hence put `false` out there. Will change and delete the redundant ENV variables.